src/capwap_ac.erl

```diff
@@ -2626,13 +2626,13 @@ add_location(NameAtom, Map) when is_atom(NameAtom) ->
     add_location(atom_to_binary(NameAtom, utf8), Map);
 add_location(Name, Map) when is_binary(Name) ->
     ?LOG(debug, "Getting location for: ~p", [Name]),
-    AVPName = capwap_config:get(ac, location_avp, undefined),
+    SessionAttr = 'IM-LI-Location',
```

Comment:
Why you try store atom inside at variable and then use this variable only once? Why not just put atom as is into map as a single key?

Reply:
True. Initially the key was going to be configurable, but there's another mapping from this key to the specific AVP in ergw_aaa, so it's probably unnecessary now.